home/templates/layouts/base.html

```diff
@@ -42,6 +42,15 @@
     <meta property="twitter:title" content="Hardhat Enterprises">
     <meta property="twitter:description" content="Hardhat Enterprises is an organisation that aims to create cyber weapons and tools that can be used to empower white-hat operations.">
     <meta property="twitter:image" content="https://hardhat.pythonanywhere.com/custom_static/assets/img/pages/home/hero-image.svg">
+
+    <!-- Content Security Policy -->
+    <meta http-equiv="X-Content-Type-Options" content="nosniff">
+<meta http-equiv="Content-Security-Policy" content="
+    default-src 'self'; 
+    script-src 'self' https://hardhat.pythonanywhere.com/ 'unsafe-inline' http://localhost:8000 https://buttons.github.io https://unpkg.com; 

```

Comment:
hardhat.pythonanywhere.com is an outdated URL
there shouldn't be a reference to localhost in here either
please see PR #123 for an example on how to correctly inject the correct URI with templating